src/main/cpp/easy/997-find-the-town-judge.h

```diff
@@ -0,0 +1,77 @@
+#ifndef LEETCODE_EASY_997_FIND_THE_TOWN_JUDGE_H_
+#define LEETCODE_EASY_997_FIND_THE_TOWN_JUDGE_H_
+
+/**
+ * Find the Town Judge
+ *
+ * In a town, there are n people labeled from 1 to n. There is a rumor that one
+ * of these people is secretly the town judge.
+ *
+ * If the town judge exists, then:
+ *
+ *   - The town judge trusts nobody.
+ *   - Everybody (except for the town judge) trusts the town judge.
+ *   - There is exactly one person that satisfies properties 1 and 2.
+ *
+ * You are given an array trust where trust[i] = [ai, bi] representing that the
+ * person labeled ai trusts the person labeled bi.
+ *
+ * Return the label of the town judge if the town judge exists and can be
+ * identified, or return -1 otherwise.
+ *
+ * @link https://leetcode.com/problems/find-the-town-judge/
+ *
+ * Solved by: Ayush Joshi <https://github.com/joshiayush>
+ */
+
+#include <cstdint>
+#include <vector>
+
+namespace leetcode {
+class FindTheTownJudge {
+ public:
+  // Finds the one single person that is the town judge.
+  //
+  // This function finds the one single person that does not trust anybody in
+  // the town but everybody trusts him/her and this person is basically judging
+  // every person around the town; to make sure that you don't mess up in front
+  // of him/her we need to find this person.
+  ::std::int64_t findJudge(
+      ::std::uint16_t n,
+      const ::std::vector<::std::vector<::std::int64_t>>& trust) {
```

Comment:
Trust size is always going to be a lot larger than the number of people we have in that town, so I don't feel like I should change `int64_t` to `uint16_t` although I can change it to `uint64_t` but keeping it just `int64_t` is not gonna have a big difference. What do you say?

Reply:
Leetcode states that the constraint on trust.size() is max 1000 elements. So I think the int64_t is overkill. But you are right, functionally the code still works.

Reply:
Anything else you would like me to change?

Reply:
For those lines specifically, prefer `std::uint16_t` over `::std::uint16_t`. [This stackoverflow post explains the difference in the syntax](https://stackoverflow.com/questions/33102207/difference-in-using-namespace-std-vs-std). I don't plan for this project to shadow the `std` namespace.